tests/integration_tests/framework/amqp_events_printer.py

```diff
@@ -73,7 +73,7 @@ def callback(ch, method, properties, body):
                 ev = json.loads(body)
                 output = cloudify.logs.create_event_message_prefix(ev)
                 if output:
-                    sys.stdout.write('{0}\n'.format(output))
+                    sys.stdout.write('{0}\n'.format(output.encode('utf-8')))
```

Comment:
this makes it print with a `b''` prefix of course, because we're formatting an .encoded bytes into a unicode (`{0}\n`)

but really, just make this `print(output)`